shared/test/test_table_coerce.rb

```diff
@@ -0,0 +1,142 @@
+require_relative 'test_helper'
+require 'table_coerce'
+
+class TablesTest < Minitest::Test
+  include SetupTest
+
+  def test_is_boolean
+    assert_equal true, TableCoerce.is_boolean?(true)
+    assert_equal true, TableCoerce.is_boolean?(false)
+    assert_equal true, TableCoerce.is_boolean?("false")
+    assert_equal true, TableCoerce.is_boolean?("true")
+    assert_equal true, TableCoerce.is_boolean?("False")
+    assert_equal true, TableCoerce.is_boolean?("True")
+
+    assert_equal false, TableCoerce.is_boolean?("asdf")
+    assert_equal false, TableCoerce.is_boolean?("1.23")
+    assert_equal false, TableCoerce.is_boolean?("1,000")
+    assert_equal false, TableCoerce.is_boolean?(0)
+    assert_equal false, TableCoerce.is_boolean?(1)
+    assert_equal false, TableCoerce.is_boolean?(1.23)
+  end
+
+  def test_to_boolean
+    assert_equal true, TableCoerce.to_boolean(true)
+    assert_equal false, TableCoerce.to_boolean(false)
+    assert_equal false, TableCoerce.to_boolean("false")
+    assert_equal true, TableCoerce.to_boolean("true")
+    assert_equal false, TableCoerce.to_boolean("False")
+    assert_equal true, TableCoerce.to_boolean("True")
+
+    [3, 'asdf'].each do |val|
+      assert_raises 'Cannot coerce to boolean' do
+        TableCoerce.to_boolean(val)
+      end
+    end
+  end
+
+  def test_is_number
+    assert_equal true, TableCoerce.is_number?(0)
+    assert_equal true, TableCoerce.is_number?(1)
+    assert_equal true, TableCoerce.is_number?(1.23)
+    assert_equal true, TableCoerce.is_number?(1000)
+
+    assert_equal true, TableCoerce.is_number?("0")
+    assert_equal true, TableCoerce.is_number?("1")
+    assert_equal true, TableCoerce.is_number?("1.23")
+    assert_equal true, TableCoerce.is_number?("1000")
+    # assert_equal true, TableCoerce.is_number?("1,000")
+
+    assert_equal false, TableCoerce.is_number?("asdf")
+    assert_equal false, TableCoerce.is_number?("true")
+    assert_equal false, TableCoerce.is_number?("false")
+    assert_equal false, TableCoerce.is_number?(true)
+    assert_equal false, TableCoerce.is_number?(false)
+    assert_equal false, TableCoerce.is_number?("123asdf")
+  end
+
+  def test_to_number
+    assert_equal 0, TableCoerce.to_number(0)
+    assert_equal 1, TableCoerce.to_number(1)
+    assert_equal 1.23, TableCoerce.to_number(1.23)
+    assert_equal 1000, TableCoerce.to_number(1000)
+
+    assert_equal 0, TableCoerce.to_number("0")
+    assert_equal 1, TableCoerce.to_number("1")
+    assert_equal 1.23, TableCoerce.to_number("1.23")
+    assert_equal 1000, TableCoerce.to_number("1000")
+    # assert_equal 1000, TableCoerce.to_number("1,000")
```

Comment:
I didn't want to have to deal with converting numbers based on locale just yet (some countries use , as a decimal separator instead of .), so I'm punting on this for now, and will track with a bug.